src/gleam/io.gleam

```diff
@@ -73,24 +75,18 @@ if javascript {
 /// ```
 ///
 pub fn debug(term: anything) -> anything {
-  debug_print(term)
+  do_inspect(term)
+  |> string_builder.to_string
+  |> println
```

Comment:
Seems we can, no idea why I overlooked that.

Also just to document that debug actually exists/runs I have added a test but only for Erlang because on JS the stdout will be polluted by the test otherwise.